File: 00-node/00-template/core/information.js
'use strict'
const properties = require('../package.json');

let information = {
    getDetails: () => {
        let details = {
            name: properties.name, 
            version: properties.version,
            author: properties.author, 
            source: properties.source,
            license: properties.license
        };
        
        return details;
    }
};

module.exports = information;

File: 02-netcore/00-template/Core/Information.cs
using _00_template.Core.Models;

namespace _00_template.Core
{
    public class Information
    {
        public InformationModel GetDetails()
        {
            var info = new InformationModel();
            info.Name = "netcore-microservices-template";
            info.Version = "0.0.4";
            info.Author = "Jersson Dongo";
            info.Source = "https://github.com/jersson/microservices-templates";
            info.License = "MIT";

            return info;
        }

    }
}